packages/mosaic/tooltip/tooltip.scss

```diff
@@ -57,36 +57,31 @@ $tooltip-distance: $tooltip-arrow-width + 3px;
 
 .mc-tooltip-arrow {
     position: absolute;
-    width: 0;
-    height: 0;
-    border-color: transparent;
-    border-style: solid;
+    width: 11.3px;
```

Comment:
why do you use float number ?

Reply:
On template the width between two corner points of the arrow is 16px. I am setting an arrow as a square and rotating it.

Reply:
It didn't solve problem, now width (or height) 15px:
![image](https://user-images.githubusercontent.com/9027254/65120466-5c959580-d9f6-11e9-9212-ba9c864efb75.png)

and if you zoom example you will see:
![image](https://user-images.githubusercontent.com/9027254/65120828-eba2ad80-d9f6-11e9-807a-cd8114d7ccf7.png)

I suggest set width and height to 12px:
![image](https://user-images.githubusercontent.com/9027254/65121061-4805cd00-d9f7-11e9-8746-17a8d764f67c.png)
and
![image](https://user-images.githubusercontent.com/9027254/65121103-58b64300-d9f7-11e9-9b26-d4d139fea493.png)

Reply:
Ok, thanks. Just a minute and I will update PR.

Reply:
Updated